tests/integration/tier0/verify-latest-kernel-check-with-failed-repoquery/test_verify_kernel_check_passes.py

```diff
@@ -0,0 +1,50 @@
+import platform
+
+import pytest
+
+
+@pytest.mark.latest_kernel_check_with_failed_repoquery
+def test_verify_latest_kernel_check_passes_with_failed_repoquery(shell, convert2rhel):
+    """
+    This test verifies, that failed repoquery is handled correctly.
+    Failed repoquery subsequently caused the latest kernel check to fail.
+    Introduced fixes should get the process past the latest kernel check.
+    """
+    get_system_release = platform.platform()
+    repofile = "broken_repo"
+    centos_custom_reposdir = "/usr/share/convert2rhel/repos/"
+
+    # Add a tainted repository with intentionally bad baseurl so the repoquery fails successfully.
+    # For CentOS we are working with hardcoded repos in /usr/share/convert2rhel/repos/centos-8.{4,5}
+    # We are skipping (at the test metadata level) the test for Oracle Linux EUS version,
```

Comment:
Should we skip this test for OL8.6 too? After we merge #607 it will be considered EUS too.

Reply:
Good point, I don't mind both. Maybe so it's more clear we can keep it as is and make it skip in #607

Reply:
Let's address all the EUS related changes in #607 
Good catch though @r0x0d !

Reply:
Seems good to me